pandas/core/arrays/numpy_.py

```diff
@@ -137,8 +138,8 @@ def __init__(self, values, copy=False):
         if copy:
             values = values.copy()
 
-        self._ndarray = values
-        self._dtype = PandasDtype(values.dtype)
+        self._ndarray: np.ndarray = values
```

Comment:
Should we move the definition to the class level as in some of the work @jbrockmendel has been doing as well? Off the top of my head I thought that was the PEP recommendation